src/test/resources/db/migration/V4__POL-550_ebs_accountnumber_to_org_id.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE org_id_latest_update (
```

Comment:
Shouldn't the file name start with `V4__POL-650`? 😃

Reply:
indeed